src/tracer/wrappers/OMP/omp-common_c.c

```diff
@@ -42,33 +42,38 @@
 	}                                                                      \
 }
 
-#if defined(PIC)
```

Comment:
Why was if defined(PIC) removed?

Reply:
Probably because we wanted the omp functions to be defined for RTEMS too, which is not PIC. Instead of deleting it I suggest to change it to "#if defined(PIC) || defined (OS_RTEMS)"

Reply:
Do functions in l. 51 to 54 exist in the runtime with these names? (__real_XXX)

Reply:
Yes, they are functions from the libgomp runtime. The "--wrap" GCC flag adds the  "__real_" prefix to the real symbols in the libgomp library.